src/Umbraco.Web.UI/Views/Partials/Grid/Editors/TextString.cshtml

```diff
@@ -4,10 +4,9 @@
 @if (Model.editor.config.markup != null)
 {
     string markup = Model.editor.config.markup.ToString();
-
     var UmbracoHelper = new UmbracoHelper(UmbracoContext.Current);
     
-    markup = markup.Replace("#value#", UmbracoHelper.ReplaceLineBreaksForHtml(Model.value.ToString()));
+    markup = markup.Replace("#value#", UmbracoHelper.ReplaceLineBreaksForHtml(TemplateUtilities.CleanForXss(Model.value.ToString())));
```

Comment:
Think you commented on the wrong line here... I'll remove it for this one:
https://github.com/umbraco/Umbraco-CMS/pull/1610/files/8bb069e996a452b2dd4fa016bca32c3ec6d933cc#diff-3215c52199fb2398f586f17d71066e82R19